runtime/jcl/common/stdinit.c

```diff
@@ -187,6 +187,18 @@ standardInit( J9JavaVM *vm, char *dllName)
 			result = (jint)initializeSystemThreadGroup(vm, (JNIEnv *)vmThread);
 			if (JNI_OK != result) {
 				goto _fail;
+			} else {
+#if JAVA_SPEC_VERSION >= 15
+				JNIEnv *env = (JNIEnv *)vmThread;
+				jclass clz = (*env)->FindClass(env, "jdk/internal/loader/NativeLibraries");
+				if (NULL != clz) {
```

Comment:
This shouldn't fall through on failure (there's an exception pending). Same for GetStaticMethodID below. Should probably `goto _fail;` in those cases.

Reply:
Fixed.